### PR TITLE
Change message for $0 invoices

### PR DESCRIPTION
**Pull request by proteusvacuum**
http://manage.dimagi.com/default.asp?163661#963251

@nickpell 
cc: @millerdev

### CODE REVIEW

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1840,12 +1840,16 @@ def email_context(self):
             subscription__subscriber__domain=self.invoice.get_domain(),
         ))
         is_small_invoice = self.invoice.balance <= SMALL_INVOICE_THRESHOLD
+        payment_status = (_("Paid")
+                          if self.invoice.is_paid or total_balance == 0
```
Comment (nickpell) [inline corehq/apps/accounting/models.py]: can't this just be ```total_balance == 0``` since that implies ```self.invoice.is_paid``` is true?

**Comment by nickpell** [comment]
no blocking comments (waiting on tests)

File: corehq/apps/accounting/models.py (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/accounting/models.py]: If there are other invoices that have a balance outstanding, `total_balance` would not be 0, but this particular invoice might be paid.

Comment (nickpell) [inline corehq/apps/accounting/models.py]: gotcha

**Comment by proteusvacuum** [comment]
tests failed, kicked travis

**Pull request merged**